doc/admin-guide/files/remap.config.en.rst

```diff
@@ -202,6 +207,30 @@ that match the second rule also match the first rule. The first rule
 takes precedence because it appears earlier in the :file:`remap.config`
 file.
 
+This is different if one rule does not have a host. For example consider these rules using the :ref:`Match-All` rule::
+
+   map / http://127.0.0.1:8001/
+	map http://example.com/dist_get_user http://127.0.0.1:8001/denied.html
+
+These rules are set up to redirect requests to another local process. Using them will result in
+
+==================================== =====================================
+Client Request                       Translated Request
+==================================== =====================================
+``http://example.com/a.gif``         ``http://127.0.0.1:8001/a.gif``
+``http://example.com/dist_get_user`` ``http://127.0.0.1:8001/denied.html``
+==================================== =====================================
+
+Note the second request is matched by the second rule even though the rules have the same base
+precedence. This is due to the first rule not matching on the host in the first phase. The first
+request is matched by the first rule because with the host it does not match the second rule and is
+therefore matched on the second phase. This will be the same even if the rules are reversed, e.g. ::
```

Comment:
How about,
In contrast, the first request does not match the second rule in the first phase (because the paths miss), but then matches the Match-All rule in the second phase.